v0.1/CSharpSwagger/CSharpSwagger/Program.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using IO.Swagger.Api;
+using IO.Swagger.Client;
+using IO.Swagger.Model;
+using Newtonsoft.Json;
+
+namespace CSharpSwagger
+{
+	class MainClass
+	{
+		public static void Main (string[] args)
+		{
+			Console.WriteLine("Hello World");
+			ApiClient client = new ApiClient("http://192.168.250.28:5003");
+			DeedApi deed_get = new DeedApi(client);
+			DefaultApi deed_post = new DefaultApi (client);
+			Console.WriteLine ("Client created");
+			Console.WriteLine(" ");
+			Console.WriteLine ("Creating Post to Client:");
+
+			IO.Swagger.Model.DeedApplication deed_app = new DeedApplication ();
+
+			deed_app.TitleNumber = "DT567568";
+			deed_app.MdRef = "e-MD123G";
+
+			Console.WriteLine ("Creating Borrower");
+
+//			{
```

Comment:
Line 21, you should be able to instantiate the DeedApplication class without explicitly declaring the "IO.Swagger.Model" since you are already using it on Line 4 ?